lib/Models/CkanCatalogGroup.js

```diff
@@ -113,12 +113,12 @@ var CkanCatalogGroup = function(terria) {
     this.useResourceName = false;
 
     /**
-     * True to allow entire WMS servers (that is, WMS resources without a clearly-defined layer) to be
-     * added to the catalog; otherwise, false.
+     * True to allow entire WMS AND WFS servers (that is, WMS/WFS resources without a clearly-defined layer) to be
+     * added to the catalog; otherwise, false. (Yes, this is poorly named.)
      * @type {Boolean}
      * @default false
      */
-    this.allowEntireWmsServers = false;
+    this.allowEntireWmsServers = false;  // and WFS servers too.
```

Comment:
It'd be cleaner to rename this.  Then add a new property named `allowEntireWmsServers` which just passes through to the new one with a `deprecationWarning` along the way.